app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/securities/ChooseFileTypeController.scala

```diff
@@ -52,7 +52,7 @@ class ChooseFileTypeController @Inject() (
 
         Ok(
           chooseFileTypePage(
-            form,
+            form.withDefault(Some(journey.answers.selectedDocumentType)),
```

Comment:
`journey.answers.selectedDocumentType` is Option already

Reply:
That's true but the form object expects an option of option which I agree is a bit strange.

Reply:
anyway, we should leave this choice empty similarly to the other journeys

Reply:
Reverted